auto-comment.yaml

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: 0BSD OR CC0-1.0
+
+repo: spdx/license-list-XML
+startAt: 2022-09-07
+pullRequests: false
+filters:
+  - type: issue
+    criteria:
+      state: closed
+      labels:
+        $and:
+          - $in: 'new license/exception: Accepted'
+          - $nin: 'Acceptance Notification Sent'
+
+actions:
+  - type: add_comment
+    body: >
+      This new license/exception request has been accepted and the information
+      for the license/exception has been added to the repository. Thank you to
+      everyone who has participated!
+
+      The license/exception will be published at https://spdx.org/licenses/ as
+      part of the next SPDX License List release, which will occur no longer
```

Comment:
Since we sometimes don't make the release dates precisely, could we soften this statement a bit?  Something like `... which is targeted to be released in 3 months or less`.  Not a big deal since the notice will go out on average 6 weeks before a release.

Reply:
@goneall, thank you for the suggestion; I've pushed an additional commit.